src/Files.App/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -402,12 +403,16 @@
 						<TextBox
 							x:Name="TileViewTextBoxItemName"
 							Grid.Row="0"
+							MaxHeight="60"
 							HorizontalAlignment="Left"
 							BeforeTextChanging="ItemNameTextBox_BeforeTextChanging"
+							ScrollViewer.VerticalScrollBarVisibility="Auto"
 							Text="{x:Bind Name, Mode=OneWay}"
+							TextWrapping="Wrap"
 							Visibility="Collapsed" />
 
 						<TextBlock
+							x:Name="ItemFileFormat"
```

Comment:
You shouldn't need to make any changes here, the changes above should be enough.

Reply:
![image](https://github.com/files-community/Files/assets/39923744/6a1c86a8-d5b5-42aa-87cf-52fe65047b45)